wpeverest/everest-forms
# Dev - Transaction Table

### All Submissions:






### Changes proposed in this Pull Request:



Previously, there are no transaction log table. This PR add transaction table in Tool.

### How to test the changes in this Pull Request:

1. Go to Everest Forms > Tool > Payment Log
2. Check whether the payment log table is visible or not.

### Types of changes:




### Other information:




### Changelog entry

> Dev - Transaction Table
